packages/@okta/vuepress-site/docs/reference/api/domains/index.md

```diff
@@ -0,0 +1,671 @@
+        },
+        {
+            "fqdn": "login.example.com",
+            "values": [
+                "${yourOktaDomain}.customdomains.okta1.com"
+            ],
+            "recordType": "CNAME"
+        }
+    ],
+    "publicCertificate": {
+        "subject": "CN=login.example.com",
+        "fingerprint": "73:68:82:7B:83:2E:48:29:A5:5E:E8:40:41:80:B3:AA:03:C4:42:43:05:73:45:BC:AA:47:00:23:A3:70:E5:C4",
+        "expiration": "2021-05-11T05:13:05.000Z"
+    },
+    "_links": {
+        "certificate": {
+            "href": "https://${yourOktaDomain}/api/v1/domains/OcDz6iRyjkaCTXkdo0g3/certificate",
+            "hints": {
+                "allow": [
+                    "PUT"
+                ]
+            }
+        },
+        "self": {
+            "href": "https://${yourOktaDomain}/api/v1/domains/OcDz6iRyjkaCTXkdo0g3",
+            "hints": {
+                "allow": [
+                    "DELETE"
+                ]
+            }
+        },
+        "verify": {
+            "href": "https://${yourOktaDomain}/api/v1/domains/OcDz6iRyjkaCTXkdo0g3/verify",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+##### Response Example (Domain not found)
+
+```http
+HTTP/1.1 404 Not Found
+Content-Type: application/json
+
+{
+    "errorCode": "E0000163",
+    "errorSummary": "Domain ID not found.",
+    "errorLink": "E0000163",
+    "errorId": "oaef65fglSvQSiUujoOXI_dAA",
+    "errorCauses": [
+        {
+            "errorSummary": "Could not find custom domain with the given id."
+        }
+    ]
+}
+```
+
+### Get All Domains
+
+<ApiOperation method="get" url="/api/v1/domains" />
+
+List all verified custom domains for the org.
+
+#### Request path parameters
+N/A
+
+#### Request query parameters
+N/A
+
+#### Request body
+N/A
+
+#### Response body
+
+The [DomainListResponse](#domainlistresponse-object)
+
+#### Usage examples
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/domains"
+```
+
+##### Response
+```json
+{
+    "domains": [
+        {
+            "id": "OcDz6iRyjkaCTXkdo0g3",
+            "domain": "login.example.com",
+            "validationStatus": "COMPLETED",
+            "_links": {
+                "self": {
+                    "href": "https://${yourOktaDomain}/api/v1/domains/OcDz6iRyjkaCTXkdo0g3",
+                    "hints": {
+                        "allow": [
+                            "GET",
+                            "DELETE"
+                        ]
+                    }
+                }
+            }
+        }
+    ]
+}
+```
+### Delete Domain
+
+<ApiOperation method="delete" url="/api/v1/domains/{id}" />
+
+Deletes your domain.
+
+#### Request path parameters
+
+Delete domain by `id`.
+
+ Parameter  | Description  | Param Type | DataType | Required |
```

Comment:
add the DataType information and Required information in the Description, please!  :)

```suggestion
 Parameter  | Type | Description |
```